rails_app/db/migrate/20230420085710_change_currencies_pk.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class ChangeCurrenciesPk < ActiveRecord::Migration[7.0]
+  def change
+    execute "ALTER TABLE currencies DROP CONSTRAINT currencies_pkey CASCADE;"
+
+    add_column :currencies, :id, :primary_key
```

Comment:
Is this really needed? Doesn't Rails assume the `id` as the PK by default?

Reply:
I think it assumes, I'm just not sure due to the overriding that were done before